Assets/Scenes/TutorialLevel.unity

```diff
@@ -4452,7 +4452,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.8223292, y: 0.20500001}
   m_AnchorMax: {x: 0.9595612, y: 0.23041128}
-  m_AnchoredPosition: {x: -0.000030517578, y: 0.39248657}
+  m_AnchoredPosition: {x: 0, y: 0.39248657}
```

Comment:
Same move comment here.

Reply:
And also in the cutscene, `RightName` this time.